homeassistant/components/ffmpeg/__init__.py

```diff
@@ -142,26 +155,28 @@ def __init__(self, hass: HomeAssistant, ffmpeg_bin: str) -> None:
         self._version: str | None = None
         self._major_version: int | None = None
 
-    @property
+    @cached_property
     def binary(self) -> str:
         """Return ffmpeg binary from config."""
         return self._bin
 
     async def async_get_version(self) -> tuple[str | None, int | None]:
         """Return ffmpeg version."""
-
-        ffversion = FFVersion(self._bin)
-        self._version = await ffversion.get_version()
-
-        self._major_version = None
-        if self._version is not None:
-            result = re.search(r"(\d+)\.", self._version)
-            if result is not None:
-                self._major_version = int(result.group(1))
+        if self._version is None:
+            if is_official_image():
+                self._version = OFFICIAL_IMAGE_VERSION
+                self._major_version = int(self._version.split(".")[0])
+            elif (
+                (version := await FFVersion(self._bin).get_version())
+                and (result := re.search(r"(\d+)\.", version))
+                and (major_version := int(result.group(1)))
+            ):
```

Comment:
The only time this check should ever run now is core installs and will only to check for and ffmpeg older than 4 :`FFmpeg release from the 4.0 release branch, which was cut from master on 2018-04-16.`